test/x509.uts

```diff
@@ -190,3 +190,15 @@ s = str(X509_CRL())
 str(X509_CRL(s)) == s
 
 
+############ Randval tests ###############################################
+
+= Randval tests : ASN1F_SEQUENCE_OF
+ASN1F_SEQUENCE_OF("test", [], ASN1P_INTEGER).randval()
```

Comment:
Could you test the returned value by specifying the seed with a call to `random.seed` before each test?